src/pages/sighting/statusCard/StatusCard.jsx

```diff
@@ -39,58 +40,69 @@ function getProgressText(intl, startDate) {
   );
 }
 
+function getStage(pipelineStep) {
```

Comment:
I think that this is beyond the scope of this ticket, but I see this getting lost if we ever need to report status in other places besides this card. I also think that there might be a way to generalize this a little more (if we end up adding more stages). But again, just pie-in-sky thoughts. Doesn't seem to be anything wrong with this current implementation.

Reply:
I'm going to leave this here, unchanged, for now because it sounds like it may still be in flux as the houston API becomes more solid.